packages/select/src/components/query-list/queryList.tsx

```diff
@@ -49,6 +49,12 @@ export interface IQueryListProps<T> extends IListItemsProps<T> {
      * Receives an object with props that should be applied to elements as necessary.
      */
     renderer: (listProps: IQueryListRendererProps<T>) => JSX.Element;
+
+    /**
+     * Whether the prop `itemRenderer` should be invoked.
```

Comment:
This comment is too implementation-specific. It should describe the intent of the prop, not its implementation.

```suggestion
     * Whether the list is disabled.
```